.circleci/config.yml

```diff
@@ -986,6 +986,7 @@ commands:
           paths:
             - /root/.local/share/pnpm/store/v3
             - /root/.cache/Cypress
+            - /root/.cargo
```

Comment:
This is where we've configured cargo to put its downloaded crates, but it doesn't do anything to cache build results. Is that your understanding?

Reply:
Yes. We'll have to use something else for that. I've started looking.